### PR TITLE
[8.0][IMP] project_categ

### PR DESCRIPTION
**Pull request by martintorresvm**
Allow to edit code field by making visible
	modified:   project_categ/__openerp__.py
	modified:   project_categ/project_categ_view.xml

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @dreispt Please have a check

**Review by elicoidal — APPROVED** [review]

File: project_categ/project_categ_view.xml [hunk 1]
```diff
@@ -12,6 +12,8 @@
                         <newline/>
                         <field name="parent_id"/>
                         <newline/>
+                        <field name="code"/>
+                        <newline/>
```
Comment (dreispt) [inline project_categ/project_categ_view.xml:16]: A note on commit message guidelines: the commit message should express the intention of the commit.
In this case you could have used something like "Fix Task Category form to show Code field" or ""Fix: show Code field on Task Category form"
And you don't need to describe the changed files - git already does that automatically.

**Review by dreispt — APPROVED** [review]

File: project_categ/project_categ_view.xml (hunk 1 above)
Comment (martintorresvm) [reply project_categ/project_categ_view.xml:16]: @dreispt Thanks I'll keep in mind for next commits.

**Comment by martintorresvm** [comment]
@elicoidal Could be merged?

**Pull request merged**